### ISSUE TITLE
Searchinator delivering incomplete results

### ISSUE THREAD

**Issue by choirboy07** [ServiceInnovationLab/FSD#185]
Searchinator is not delivering comparable results to the Family Services Directory

**Comment by choirboy07** [comment]
here is an example where a provider in the FSD data is not coming up in the Searchinator results yet it is within 300 metres of a provider that does.

[How is Searchinator searching.docx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5c61fc6e865e1b0011e12374/125516c66e82c728ace21e0d46c09b8d638b90e9bcecbe51a755f99a285228616917a62a6286eb21b145394d4d1f0cf31b0d0c00eef39b2ef5b73473d85458bb82685d7c903d11aaead11fb609e821995397f4edd4189fb5646fb2c99d452c1f0d1288fa5837a59942cf5bfe4ee64e47e9.docx)

**Comment by CharlotteHinton** [comment]
@choirboy07 We are seeing a bit of a discrepancy between the doc you've sent and what we are seeing online here.

With our searchinator - we see 6 results
On your website we are seeing - 21 results (11 local)

The document you've provided says there should be 7. Can you let us know if that's a typo or could also send through a screenshot of what you are seeing, as that will be really helpful

**Comment by choirboy07** [comment]
Hi, I have 7 in the document as they have a Gisborne City  Street address - the other 4 on the main Directory are Ruatoria addresses, which is 100+kms from Gisborne City.

**Comment by CharlotteHinton** [comment]
@dlouise64 @JacOng17 feedback from @choirboy07 for you.

**Comment by sarahneeson** [comment]
in case it's useful, I've done a comparison between Find Whanau Services (Searchinator) and Family Services Directory (FSD):
[Searchinator vs FSD.xlsx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a2613c599d956001145c2b3/125516c66e82c728ace21e0d46db919b31818ba081e8ab4cb61de687696014576b48e13b628ae92dbc56395b4d5a1dad495b0d4cafafda2ea4be6a708a5a59bed1645726c72f0dbefad341eb5ee779c11ac7a4b6984dc7e23c20e28987463e1c051980fc5927ed.xlsx)

The TL;DR is: I can explain the difference in results between the two sites for 4/6 results, but there are 2 results where I can't explain why they aren't showing in Searchinator.

Those results are for:
Provider 7292 Open Home Foundation (OHF) - Wairarapa, and
Provider 5303 Rangitane O Wairarapa Incorporated

**Comment by choirboy07** [comment]
I have done some testing and issue appears to be where the physical address has no numbers (Street or Post code) that can be reference to match the location.  This appears to be for any physical address and not just those that start with Cnr - see last entry on spreadsheet. This issue also relates to card 201.   Note I am not sure why Stand Children's Services for Manukau is not coming up in keyword search only.
[Searchinator Location Searching .xlsx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5c61fc6e865e1b0011e12374/125516c66e82c728ace21e0d46db919b31818ba081e8ab4cb61ddc9b2a47337a260ae3187584f12bb74b75064f4d52be011a001ea9aacd39aca439238b5408bfd93e0927c33f19fab88219ed52ba29831d84a4a0980e91b6697abad2c51a7c0c0f0a88f85734a59c49c15cf44ce94851a611242e.xlsx)

**Comment by sarahneeson** [comment]
@lamorrison  the summary of what we've found when looking specifically for  Open Home Foundation (OHF) - Wairarapa (provider 7292) is:

1.  Enter topic or organisation: [ open home ] - result set includes the provider

2. Enter address [ Masterton ] - result set includes the provider

3. Enter address [ 2 Akura Road, Masterton 5810] - result set includes the provider

4. Enter address [340 Queen Street, Masterton 5810] - result set DOES NOT include the provider

So the address search can identify that Open Home Wairarapa is in Masterton, and it can identify it being close to another street in Masterton, but for some reason it can't identify that it's in close proximity to Queen Street Masterton.

Not sure what the likely cause is, but in case it's helpful... !

**Comment by lamorrison** [comment]
I have made a separate card for the bug relating to Stand Children's Services, #216

**Comment by lamorrison** [comment]
Hi @sarahneeson , I tried the search you mentioned for OHF Wairarapa today and it seems to be working for me. Could you confirm if it's still missing from the 340 Queen Street search for you?
![Screen Shot 2019-03-22 at 13.55.59.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5c464bf6bf6914001135786b/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5700b77206456f16b7191a379ec0c2e54415a47be5d4a1f06a3a3dc78e2a436628f0f0ae5d23f5727906e14aeb0d41fbf07e8738b468de4bd9850d2aa6c7bb3d4d216744b470a83ed5c3caa984ecd5bf44ae74f50ac1a3e60f43b.png)

**Comment by sarahneeson** [comment]
Hi @lamorrison  - Yep, it's still missing.  Apologies, I should have specified it was in combination with the category Parents and Caregivers.

Here's the search where it doesn't appear:
https://serviceinnovationlab.github.io/FSD/#/FSD/?address=340%20Queen%20Street%2C%20Masterton%205810&category=Parents%20and%20Caregivers&latitude=-40.9540232833&longitude=175.65468015&radius=50

**Comment by choirboy07** [comment]
Hi, I've attached a table to show search outcomes for the same search constructs using 340 Queen Street and then Masterton.  I haven't figured out the why yet but it does highlight differences in results between the 2 "Addresses".
[Open Home Searches Table.xlsx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5c61fc6e865e1b0011e12374/125516c66e82c728ace21e0d46c7849f2dc2aba682ecff70a15ce297214334331d05a12775c7af6aaf50740b0a0c0abe4b4a5a17fdf18a79aee56a238f5e03bbd16d5926953d18a4ea830da313f871d51bdaebe5c01982fa3737e68c854e3b1b061a8bfd5e3ca89a4bcf4abb.xlsx)

**Comment by dlouise64** [comment]
@lamorrison @JacOng17 I bumped the limit to 500 now I can see "Open Home Foundation (OHF) - Wairarapa". Ready for dev testing.

**Comment by sarahneeson** [comment]
Two of the previous searches that weren't showing Open Home Foundation (OHF) - Wairarapa in the results are now showing that provider as expected.  
I still have a question about the placement of this provider in one of the searches, but will raise this question in a separate issue.  

![Retesting Open Home Wairarapa searches](https://user-images.githubusercontent.com/34259860/56004623-71238280-5d20-11e9-882e-21ceef9fb5c0.PNG)
